Package.swift

```diff
@@ -5,28 +5,37 @@ import PackageDescription
 let package = Package(
     name: "swift-webdriver",
     products: [
-        .library(name: "WebDriver", targets: ["WebDriver", "WinAppDriver"]),
+        .library(name: "WebDriver", targets: ["WebDriver"]),
     ],
     targets: [
         .target(
             name: "WebDriver",
             path: "Sources/WebDriver"),
-        .target(
-            name: "WinAppDriver",
-            dependencies: ["WebDriver"],
-            path: "Sources/WinAppDriver"),
         .target(
             name: "TestsCommon",
             path: "Tests/Common"),
-        .testTarget(
-            name: "WinAppDriverTests",
-            dependencies: ["TestsCommon", "WebDriver", "WinAppDriver"],
-            // Ignore "LNK4217: locally defined symbol imported" spew due to SPM library support limitations
-            linkerSettings: [ .unsafeFlags(["-Xlinker", "-ignore:4217"]) ]),
         .testTarget(
             name: "UnitTests",
-            dependencies: ["TestsCommon", "WebDriver", "WinAppDriver"],
+            dependencies: ["TestsCommon", "WebDriver"],
             // Ignore "LNK4217: locally defined symbol imported" spew due to SPM library support limitations
-            linkerSettings: [ .unsafeFlags(["-Xlinker", "-ignore:4217"]) ]),
+            linkerSettings: [ .unsafeFlags(["-Xlinker", "-ignore:4217"], .when(platforms: [.windows])) ]),
     ]
 )
+
+#if os(Windows)
```

Comment:
This prevents cross-compilation :(.  This is evaluated in the context of the build, not the host.

Reply:
Fair enough.  Does spm support cross compilation today?  

We don't support it currently.  This gets us closer to our xplat goals.  I'm happy to iterate if cross compilation is possible.